tests/unittests/OperatorTest.cpp

```diff
@@ -1648,6 +1649,47 @@ TEST_P(InterpOnly, IntLookupTable) {
   }
 }
 
+/// Check that the sequence of extract-batchedadd-concat works.
+TEST_P(Operator, testBatchAdd) {
+  unsigned numSlices = 10;
+  auto *input =
```

Comment:
can you add a test for quantized batchAdd for completeness

Reply:
Done.